authors/apps/articles/tests/base_test.py

```diff
@@ -0,0 +1,47 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+
+class TestBaseCase(APITestCase):
+    def setUp(self):
+        self.signup_url = reverse('authentication:register_user')
+        self.login_url = reverse('authentication:login_user')
+        self.create_list_article_url = reverse('articles:articles')
+        self.test_user = {
+                'username': 'Test',
+                'email': 'test@user.com',
+                'password': 'Password12#'
+            }
+        self.article = {
+                "title": "Test Article",
+                "description": "Test description for the article",
+                "body": "Test body for the article",
+                "author": 1
+        }
+
+    def signup_user(self):
+        return self.client.post(self.signup_url,
+                                self.test_user,
+                                format='json')
+
+    def login_user(self):
+        self.signup_user()
+        response = self.client.post(self.login_url,
+                                    self.test_user,
+                                    format='json')
+
+        token = response.data['token']
+        return token
+
+    def create_article(self):
+        token = self.login_user()
+        url = self.create_list_article_url
```

Comment:
I don't the reassignment of `self.create_list_article_url` to `url` is necessary just use `self.create_list_article_url` directly.